lib/fastlane/plugin/wpmreleasetoolkit/helper/deprecated.rb

```diff
@@ -0,0 +1,52 @@
+module Fastlane
+  module Helper
+    # A helper class to store deprecated methods and actions
+    class Deprecated
+      # Creates a project_root_folder Fastlane ConfigItem
+      #
+      # @return [FastlaneCore::ConfigItem] The Fastlane ConfigItem for the `PROJECT_ROOT_FOLDER` environment variable
+      #
+      def self.project_root_folder_config_item
+        verify_block = proc do
+          unless ENV['PROJECT_ROOT_FOLDER'].nil?
```

Comment:
IINM, doesn't this block only run if `params[:project_root_folder]` is requested?

ie – do we need the `unless` wrapper?

Reply:
Good point, fixed in https://github.com/wordpress-mobile/release-toolkit/commit/bd6b643d381f1fd5998af86b677a147d6975f63a